Scripting-Engine/Scripting-Engine/LeagueSandbox-Default/Champions/Ezreal/E.cs

```diff
@@ -22,54 +23,44 @@ static void onFinishCasting(Champion owner, Spell spell)
             Vector2 trueCoords;
 
             if (to.Length() > 475) {
-                //to = to.Normalize()
-                //local range = to * 475
-                //trueCoords = current:copy() + range
-            } else {
-                trueCoords = new Vector2(spell.X, spell.Y);
-             }
-            /*
-
-    if to:length() > 475 then
-        to = to:normalize()
-                local range = to * 475
-                trueCoords = current:copy() + range
+                to = Vector2.Normalize(to);
+                Vector2 range = to * 475;
+                trueCoords = current + range;
+            }
             else
-        trueCoords = Vector2:new(spell.X, spell.Y)
-            end
-        
-    addParticle(owner, "Ezreal_arcaneshift_cas.troy", owner.X, owner.Y);
-            teleportTo(owner, trueCoords.x, trueCoords.y)
-            addParticleTarget(owner, "Ezreal_arcaneshift_flash.troy", owner);
-
-            local target = nil
-            local units = getUnitsInRange(owner, 700, true)
-        
+            {
+                trueCoords = new Vector2(spell.X, spell.Y);
+            }
```

Comment:
See the comment later on regarding the logic for calculating the target with range/center.